src/converter-property-provider.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Sarif from './sarif/sarif-2.1.2';
+
+export function getConverterProperties(): Sarif.Run['conversion'] {
+    return {
+        tool: {
+            driver: {
+                name: 'axe-sarif-converter',
+                fullName: 'axe-sarif-converter v1.3.0',
```

Comment:
Do we expect to eventually have an automated way to update the version number here as we generate new builds of the converter?